<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FetchOctopartClass</class>
 <widget class="QWidget" name="FetchOctopartClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FetchOctopart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search Parameters</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRunRequest">
        <property name="text">
         <string>Run Request</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetResults"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonRunRequest</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>989</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
